### PR TITLE
Colors hardening

### PR DESCRIPTION
**Pull request by gregaubert**
Relates to https://github.com/SonarSource/sonarcloud-core/pull/157

### CODE REVIEW

**Comment by siegfried-ehret-sonarsource** [comment]
That gives us how many shades or red ? :)

File: CHANGELOG.md
Comment (jeremy-davis-sonarsource) [inline CHANGELOG.md]: Include ticket numbers?

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM! Can you wait for my PR so we do a single release?

**Comment by gregaubert** [comment]
Yeah sure, I wasn't planing on releasing this anyway thinking that is was not worthy enough alone. So I'll merge and let you do the release when you are done with your PR.

**Pull request merged**
